microsoft-edge/devtools-guide-chromium/customize/extensions.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Extend DevTools with extensions
+description: Learn to build a Microsoft Edge extension that extends DevTools.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 09/30/2022
+---
+# Extend DevTools with extensions
+
+If the tools available in Microsoft Edge DevTools are not enough for your use cases, you can add new tools by installing Microsoft Edge extensions or building your own extensions.
```

Comment:
```suggestion
If the tools available in Microsoft Edge DevTools aren't enough for your use cases, you can add new tools by installing Microsoft Edge extensions or building your own extensions.
```
per style guide & probably Acrolinx.  Consider running Acrolinx on this .md file.